tests/engine/storage/test_storage.py

```diff
@@ -47,10 +47,12 @@ def test_metadata_serialization(domain: Domain, tmp_path: Path):
     trained_at = datetime.utcnow()
     rasa_version = rasa.__version__
     model_id = "some unique model id"
+    assistant_id = "test_assistant"
     metadata = ModelMetadata(
         trained_at,
         rasa_version,
         model_id,
+        assistant_id,
```

Comment:
We need to test if this is set as we are currently doing for rest of parameters.

Reply:
Addressed: https://github.com/RasaHQ/rasa/pull/12019/commits/6ece56025282dbcb0c62b3136be33cef96566f16